### PR TITLE
Issue 362: Trimmer README

### PR DESCRIPTION
**Pull request by pearsonca**
## Description

This PR does more against #362 on the trimming / dropdowns fronts, as well as addressing some github-flavored-markdown issues with parsing README.Rmd.

## Checklist

- [X] My PR is based on a package issue and I have explicitly linked it.
- [X] I have included the target issue or issues in the PR title in the for Issue(s) *issue-numbers*: PR title
- [X] I have read the [contribution guidelines](https://github.com/epinowcast/epinowcast/blob/main/CONTRIBUTING.md).
- [X] I have tested my changes locally.
- [X] I have added or updated unit tests where necessary.
- [X] I have updated the documentation if required.
- [X] My code follows the established coding standards.
- [ ] I have added a news item linked to this PR.
- [x] I have reviewed CI checks for this PR and addressed them as far as I am able.

### CODE REVIEW

**Comment by pearsonca** [comment]
Alright, ready for a first pass - I should highlight a lot of the +lines are structural carriage returns, elements, etc. It is actually trimmer.

Some notes:
 - loooooots of guess wording here.
 - I think we should consolidate a bit on where we direct people, both for their and our sakes, particularly for Q&A. Seems like the community forum is better place for that, but we could also simply consolidate all the foruming to github discussions.

File: README.Rmd [hunk 1]
```diff
@@ -30,67 +30,109 @@ license](https://img.shields.io/badge/License-MIT-blue.svg)](https://github.com/
...
+_Nowcasting_ is predicting the *current* situation, addressing the practical reality that the data we want *now* will not be available until *later*. The `{epinowcast}` package provides you tools for nowcasting epidemiological data in a convenient, flexible, and efficient collection of `R` functions. Those functions implement a model of infectious disease spread and reporting using a semi-mechanistic Bayesian approach with a range of reporting and generative processes. Using this approach, you will typically see changes in trends earlier and more reliably than if using methods like truncation.
```
Comment (pearsonca) [inline README.Rmd:33]: big [citation needed] energy on that truncation bit at the end. i don't doubt its true, but readers definitely should.

Comment (seabbs) [reply README.Rmd:33]: Hmm yes agree. As this first paragraph is also the package description can we move this edit to a new issue and discuss as its own PR?

Comment (seabbs) [reply README.Rmd:33]: (note I quite like the changes but think it needs wider discussion/consideration)

**Review by seabbs — COMMENTED** [review]
This is really nice - thanks @pearsonca! 

My only major comments are that I think we might need to workshop the first paragraph in its own issue as it's such a key gateway to the package and is also the description at the moment and that we should consider no longer rendering the citation and instead just point to the R citation call and the citation.cff

File: README.Rmd (hunk 1 above)
Comment (pearsonca) [reply README.Rmd]: reverted in this, extracted as issue #393.

**Comment by pearsonca** [comment]
> My only major comments are that I think we might need to workshop the first paragraph in its own issue as it's such a key gateway to the package and is also the description at the moment

done, see #393.

> and that we should consider no longer rendering the citation and instead just point to the R citation call and the citation.cff

I've tried yet-another-option in latest rev, but zero pride-of-authorship on this item.

Lastly, how do you want to approach the consolidation-of-fora Q? I don't think it needs to be solved now, but it probably could be - there's no update on the github discussions Q & A, so we could just start pointing to the community area (and probably rename it, pin it to displayed topics, etc).

**Comment by pearsonca** [comment]
aside: regarding the PR checklist, I think this can fall against the existing NEWS item (much like still using the closed issue).

File: README.Rmd [hunk 2]
```diff
@@ -30,67 +30,109 @@ license](https://img.shields.io/badge/License-MIT-blue.svg)](https://github.com/
...
 
-If interested in expanding the functionality of the underlying model note that `epinowcast` allows users to pass in their own models meaning that alternative parameterisations, for example altering the forecast model used for inferring expected observations, may be easily tested within the package infrastructure. Once this testing has been done alterations that increase the flexibility of the package model and improves its defaults are very welcome via pull request or other communication with the package authors. Even if not wanting to add your updated model to the package please do reach out as we would love to hear about your use case.
+If you need a different underlying model for you work: `epinowcast` lets you pass your own models! If you do try new model parameterisations that expand the overall flexibility or improve the defaults, please let us know either here or on the [community forum](https://community.epinowcast.org/). We always like to hear about new use-cases, whether or not they are directed at the core `epinowcast` applications.
```
Comment (pearsonca) [inline README.Rmd:135]:
```suggestion
If you need a different underlying model for your work: `epinowcast` lets you pass your own models! If you do try new model parameterisations that expand the overall flexibility or improve the defaults, please let us know either here or on the [community forum](https://community.epinowcast.org/). We always like to hear about new use-cases, whether or not they are directed at the core `epinowcast` applications.
```

**Comment by pearsonca** [comment]
Weird that benchmarking triggered on this - due to DESCRIPTION file spacing fixes?

**Review by seabbs — APPROVED** [review]

**Comment by pearsonca** [comment]
> This is great @pearsonca - thanks. All looks good to me. Note I removed the printing of the citation as it can get out of date. Are you happy with that change (just confirm and then this can go in the merge queue or we can discuss).

Fine with that - merge queue away!

**Pull request merged**